examples/bri-3/Logger.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common';
```

Comment:
this file is now obsolete and can be removed completely

Reply:
Updated